### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Employee Experience Team, 21-526EZ, Max CFI

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#74323]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Sorry for the late feedback! I was out of office for a sick kid, and it's taken a couple of days to catch up.
- Love the CodePen prototypes. I wish more teams did this, it's fantastic.
- All of this looks really good! I like both options and have no major concerns, just some minor feedback.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** The heading structure in the prototypes doesn't quite match what a live form on VA.gov is going to look like. "Rated Disabilities" should be an H3 (beneath the step indicator H2), and the accordions in the yellow prototype should each be H3s as well.
- **Must:** Super small, the `sr-only` text on the purple prototype's link to the FAQ page is a little different than what's used elsewhere on VA.gov (I believe it's just "opens in a new tab").
- **Something to be aware of:** Focus management in your CodePen prototypes is a little different than in the live VA forms system, which might be something the screen reader users you test with mention. I don't think it's necessary to be an exact match for your testing, but you might get a "this is different than I expect" from someone. The big things I'm noticing are:
   - Transitions between pages in the form, on VA.gov focus goes straight to the step indicator H2 when you land on the new page.
   - If a user tries to Continue without making a selection, on VA.gov focus goes to the question with the error
   Easy things to miss as a sighted user, but definitely things screen reader users may notice.

**Comment by carodew** [comment]
For my own tracking:

- [ ] Must: The heading structure in the prototypes doesn't quite match what a live form on VA.gov is going to look like. "Rated Disabilities" should be an H3 (beneath the step indicator H2), 
  - [ ] and the accordions in the yellow prototype should each be H3s as well.
- [ ] Must: Super small, the sr-only text on the purple prototype's link to the FAQ page is a little different than what's used elsewhere on VA.gov (I believe it's just "opens in a new tab").
- [ ] Something to be aware of: Focus management in your CodePen prototypes is a little different than in the live VA forms system, which might be something the screen reader users you test with mention. I don't think it's necessary to be an exact match for your testing, but you might get a "this is different than I expect" from someone. The big things I'm noticing are:
  - [ ] Transitions between pages in the form, on VA.gov focus goes straight to the step indicator H2 when you land on the new page.
  - [ ] If a user tries to Continue without making a selection, on VA.gov focus goes to the question with the error
  - [ ] Easy things to miss as a sighted user, but definitely things screen reader users may notice.

**Comment by carodew** [comment]
We've made all these changes to the prototype. Worth nothing that with CodePen the overall navigation will still be slightly different (apparently it's all in an iframe) but we can set expectations with the participant that the navigation will be slightly different than what they're used to on va.gov.
